packages/esm-patient-registration-app/src/patient-registration/field/obs/obs-field.component.tsx

```diff
@@ -170,8 +171,8 @@ function DateObsField({ concept, label, required, placeholder }: DateObsFieldPro
   const { setFieldValue } = useContext(PatientRegistrationContext);
 
   const onDateChange = useCallback(
-    (date: Date) => {
-      setFieldValue(fieldName, date);
+    (birthdate: CalendarDate) => {
```

Comment:
A small observation; this date can be any date so naming it birthdate is not fully representative of what it does.

Reply:
You are absolutely right, that was a grave error on my part. Thanks for catching it!